src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -376,7 +377,12 @@ class GoogleDataprocDAO(protected val leoServiceAccountEmail: WorkbenchEmail,
 
   /* set the notebook service account as the staging bucket owner */
   override def setStagingBucketOwnership(cluster: LeoCluster): Future[Unit] = {
-    cluster.serviceAccountInfo.notebookServiceAccount match {
+    setBucketOwnership(cluster.serviceAccountInfo.clusterServiceAccount, cluster.googleProject, cluster.clusterName)
+    setBucketOwnership(cluster.serviceAccountInfo.notebookServiceAccount, cluster.googleProject, cluster.clusterName)
+  }
+
+  private def setBucketOwnership(serviceAccount: Option[WorkbenchEmail], googleProject: GoogleProject, clusterName: ClusterName): Future[Unit] = {
```

Comment:
nit: new name is a bit misleading here

Reply:
Yeah we could keep the name `setStagingBucketOwnership`?